spec/translated_properly_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+RSpec.describe Card, :type => :model do
+  describe('original_text not equal translated_text') do
+    it 'should compare two words and add error if they equal' do
+      card = Card.create(original_text: "ДоМ", translated_text: "дом")
+      expect(card.errors[:translated_text]).to include "words must be uniq"
+    end
+  end
+
+  describe ('review date must increased') do
+    it 'inreasing date' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.